app/models/concerns/audit_associations.rb

```diff
@@ -1,15 +1,11 @@
 module AuditAssociations
-  def self.included(base)
-    base.send :include, InstanceMethods
```

Comment:
This is pointless, we can just include the module directly (or prepend, in this case, since we want to override the `audited_attributes` method)